io.vikunja.Desktop.appdata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.vikunja.Desktop</id>
```

Comment:
please don't use `Desktop` and the last component. It is confusing and non descript.

Reply:
`Vikunja` would be appropriate.

Reply:
@kolaente you could name it `io.vikunja.App`

Reply:
this is differently bad taste. At least it's not `Desktop`.